### ISSUE TITLE
Consider using official packages

### ISSUE THREAD

**Issue by jlecour** [DavidWittman/ansible-redis#116]
Hi,

I've started using your role (thank your for sharing your work), and I've noticed that it is compiling Redis from sources. It's great if you want to have the latest (or a specific) version of it, but not so great when you want the packaged version from the distribution.

I've noticed that the [Elasticsearch role](https://github.com/elastic/ansible-elasticsearch) is offering the possibility to use the package from the official repository, use a downloaded package or compile from source. It's definitely more complex to handle, but very good for the user, who is able to choose how he wants things to be installed.

Thanks again for sharing this :bow:

**Comment by DavidWittman** [comment]
@jlecour Agreed. I'll look into this. It gets a little hairy because there are differences in the packages between distributions too, but I can at least give a shot at baseline support for RHEL and Debian.

**Comment by DavidWittman** [comment]
https://github.com/ANXS/nginx/ does this too via the `nginx_install_method` variable.

**Comment by retr0h** [comment]
Any news on this?  Would be helpful to install from package vs needing to compile.

**Comment by DavidWittman** [comment]
No news from my end, but I'd accept a PR which implements this via `redis_install_method`.

**Comment by smutel** [comment]
Hello,

I am interested to this feature. Could you please guide me ?

I don't find this parameter in the master branch. Is-it already merged ?
In addition I didn't find anything related to https://github.com/pieterlexis/ansible-redis/commit/8159f9f4fd945cb6e9a64e02bbf96f218e86df3c in the master ?

Thanks.
